integrationtests/src/test/java/org/axonframework/integrationtests/queryhandling/AbstractSubscriptionQueryTestSuite.java

```diff
@@ -227,7 +215,6 @@ void testCompletingSubscriptionQueryExceptionally() {
         // when
         SubscriptionQueryResult<QueryResponseMessage<List<String>>, SubscriptionQueryUpdateMessage<String>> result =
                 queryBus.subscriptionQuery(queryMessage,
-                                           new SubscriptionQueryBackpressure(FluxSink.OverflowStrategy.IGNORE),
                                            Queues.SMALL_BUFFER_SIZE);
```

Comment:
We should add new tests for leave tests for deprecated methods until those methods are removed.

Reply:
I can copy paste all old tests that used Backpressure strategy?